src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -258,7 +262,9 @@ def preprocess(self, inputs, chunk_length_s=0, stride_length_s=None):
             # XXX: Carefuly, this variable will not exist in `seq2seq` setting.
             # Currently chunking is not possible at this level for `seq2seq` so
             # it's ok.
-            align_to = self.model.config.inputs_to_logits_ratio
+            # If this variable is not available we cannot know how to align
+            # properly. timestamps are likely to be wrong.
+            align_to = getattr(self.model.config, "inputs_to_logits_ratio", 1)
```

Comment:
`align_to` is really important for wav2vec2&co since we have X input data values for 1 logits output value, so we **must** align if we don't want to have spurrious replication of letters because we chunked WITHIN a wav2vec2 chunk.

Here since everything is already muddled, it's OK to say 1, but we're also loosing that property.
Since the muddling in `feature_extractor` is likely to be bigger, we're already going to have more issues when stiching back together than this (here we're talking outputting "woorld" instead of "world" which is incorrect but might be acceptable).